loki/transformations/single_column/tests/test_scc.py

```diff
@@ -315,7 +323,10 @@ def test_scc_annotate_openacc(frontend, horizontal, blocking, acc_data):
 
     # Ensure routine is anntoated at vector level
     pragmas = FindNodes(Pragma).visit(kernel.ir)
-    assert len(pragmas) == 5
+    if HAVE_FP:
```

Comment:
Optional: The "no FP" case isn't covered in the testing at the moment - you could of course fiddle it in by temporarily setting `HAVE_FP` to False, but you could instead also mark this test as `skipif` or `xfail` if fparser isn't available.